src/components/CreateEnvironment/index.tsx

```diff
@@ -7,12 +7,15 @@ import { useState } from 'react';
 import { ALL_PACKAGES, CREATE_ENV } from '../../queries';
 import { PackageContext } from './PackageContext';
 import { PopUpDialog } from './PopUpDialog';
+import { useLocalStorage } from '@uidotdev/usehooks';
 
 // CreateEnvironment displays the 'create environment' page.
 export default function CreateEnvironment() {
   const { loading, data, error } = useQuery(ALL_PACKAGES);
   const [envBuildError, setEnvBuildError] = useState("");
   const [envBuildSuccessful, setEnvBuildSuccessful] = useState(false);
+  const [, setIgnoreReady] = useLocalStorage("environments-ignoreready", false);
+	const [, setOnlyMine] = useLocalStorage("environments-mine", false);
```

Comment:
What happened to the formatting here?

Reply:
should be fixed now; not a clue what caused the tab there